content/post/writing-a-book-remotely.md

```diff
@@ -0,0 +1,576 @@
+
+We also made sure every co-author works independently on their own area (like OAuth2 login or RDS setup) to avoid any misunderstanding.
+
+Everybody was working on a dedicated chapter so there wasn't any conflict or idle time due to waiting on some else input. Writing the new chapter consisted both of doing some research, adjusting the sample application and writing the manuscript.
+
+As were incrementally releasing new version of th eBook, we tried to work on the modules and chapters in sequence. It should be more or less and append-only of chapters and no inline adding of chapters so that the readers that get a copy early, can continue reading the book were they left if there's a new version available.
+
+In parallel to our first release preparation, we create a basic [landing page](https://stratospheric.dev) and signed up for a mailing list provider (Convertkit back then). To keep things simple, we bought an HTML Bootstrap theme and used GitHub pages to quickly launch this landing page and add the mailing list signup form to allow potential readers get more information about our progress.
+
+As there wasn't any earnings, we kept the costs low to only invest more $ once we see and adoption.
+
+This signup form also acted as a first litmus test for us to see how many developers are interested in bringing this book over the finish line.
+
+Before we could release a first version, we had to finalize our eBook title, find a logo and a cover.
+
+We started with the placeholder project name `aws101`. However, this was clearly violating trademarks of AWS, we had to find another name. As the book is all about getting ready for the cloud and effectively deploy applications to production in the cloud, the symbol of a starting rocket seemed natural.
+
+After some brainstorming for possible names, we ended up with: `Stratospheric`.
+
+We ordered two design examples for the logo on Fiverr and picked the best one:
+
+![Stratospheric eBook Cover](/img/stratospheric-ebook/stratospheric-ebook-cover.png#center "Stratospheric eBook Cover")
+
+On the 08 of November 2020, we released the first manuscript. From this day forward it took us approximately two months for the next revision to be published.
+
+We started with a base price of $X. Keep in mind that we get 80% royalties but have to split those equally among all authors.
+
+## Iterate For 6-7 Month ✍🏻
+
+What followed where multiple months of meeting every two weeks, discussing the next steps and writing the book. This step-by-step approach gave us the ability to react early on reader feedback and stay motivated as we could release something to the world sometimes.
+
+I've used this same iterative approach with an [early bird audience for my first online course](https://progmot.com/post/creating-and-selling-my-first-online-course/).
+
+While you can have the best eBook or product, if your potential readers and customers are not getting in front of it.
```

Comment:
```suggestion
While you can have the best eBook or product, if your potential readers and customers are not getting in front of it then it may be a waste of time.
```